packages/web/src/pages/pay-and-earn-page/components/SalesTable.tsx

```diff
@@ -169,6 +175,9 @@ export const SalesTable = ({
       scrollRef={scrollRef}
       fetchBatchSize={fetchBatchSize}
       wrapperClassName={styles.tableWrapper}
+      tableHeaderClassName={
```

Comment:
nit: instead of adding conditional top padding, why not add bottom margin to the network take rate copy itself?

Reply:
Personally prefer not to use margin unless we absolutely have to in favor of internal padding for components and gap settings on flex containers.

Reply:
these tables have a padding by default, so I'd still have to remove it